### ISSUE TITLE
Расчет Accuracy перенести в AccuracyCallback

### ISSUE THREAD

**Issue by ildar-ceo** [bayrell/tiny_ai_helper#5]
```
ai.utils.fit(
    model, train_dataset, val_dataset,
    batch_size=32, epochs=10,
    callbacks=[
        ai.AccuracyBinaryCallback(),
        ai.ProgressCallback(one_line=False, progress_iter=True, show_lr=True),
        ai.SaveCallback(),
    ]
)
```
